README.md

```diff
@@ -1,24 +1,85 @@
-# operator-k8s
+# Canonical Livepatch Server (K8s Charm)
+
+[![CharmHub Badge](https://charmhub.io/canonical-livepatch-server-k8s/badge.svg)](https://charmhub.io/canonical-livepatch-server-k8s)
+[![Release](https://github.com/canonical/livepatch-k8s-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/livepatch-k8s-operator/actions/workflows/publish_charm.yaml)
+[![Tests](https://github.com/canonical/livepatch-k8s-operator/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/canonical/livepatch-k8s-operator/actions/workflows/test.yaml?query=branch%3Amain)
 
 ## Description
 
-TODO: Describe your charm in a few paragraphs of Markdown
+The Livepatch K8s charm is the easiest and the recommended way to deploy the Livepatch server on K8s. This charm configures and runs the Livepatch server, which serves Livepatch-es and metadata attached to them to the clients. Canonical Livepatch patches high and critical linux kernel vulnerabilities, removing the immediate need to reboot to upgrade the kernel, instead allowing the downtime to be scheduled. It is a part of the Ubuntu Pro offering.
+
+⚠️ For users who want to deploy an entire Livepatch on-prem server (including its dependencies), it is recommended to use the `k8s/stable` channel of the [bundle](https://charmhub.io/canonical-livepatch-onprem?channel=k8s/stable) made for this purpose. For more detailed steps on using the bundle, please see the [tutorials](https://ubuntu.com/security/livepatch/docs/livepatch_on_prem/tutorial) on the Livepatch website.
```

Comment:
The tutorial link for k8s specifically here https://ubuntu.com/security/livepatch/docs/livepatch_on_prem/tutorial/Getting%20started%20with%20Livepatch%20and%20Microk8s

Reply:
I've noticed it, but I'm a bit uneasy with putting a concrete link, because if we change the title of the Discourse topic (which has happened before), then the link will be broken. I think it's best to point them to the directory and let them pick the (currently, only one) topic.

Reply:
Makes sense.